dags/openshift_nightlies/tasks/benchmarks/e2e.py

```diff
@@ -61,6 +61,21 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease, task_group
                 "ORCHESTRATION_USER": self.config['provisioner_user'],
                 "ORCHESTRATION_HOST": self.config['provisioner_hostname']
             }
+        elif self.release.platform == "aws":
```

Comment:
this will only add the var for aws , if you want to add it for aws,gcp and azure for chaos, you could have an if condition like `if self.release.variant == "chaos":`

Reply:
better yet just make a new task class for chaos benchmarks since this class is called `E2EBenchmarks` because it's for benchmarks specifically from https://github.com/cloud-bulldozer/e2e-benchmarking